src/commit.rs

```diff
@@ -87,3 +87,17 @@ pub fn pre_commit_check(pre_commit_command: Option<String>, message: &str) -> Re
     }
     Ok(())
 }
+
+pub fn git_add_all_modified() -> Result<()> {
+    let output = git_exec(&["add", "-u"])?;
+    std::io::stdout().write_all(&output.stdout)?;
+    std::io::stderr().write_all(&output.stderr)?;
+
+    if !output.status.success() {
+        return Err(anyhow!(
+            "Failed to get git path. Make sure you are in a git repository"
```

Comment:
Actually, the error would be something like: "Could not add files to staged area"